wikidict/render.py

```diff
@@ -267,6 +267,7 @@ def parse_word(word: str, code: str, locale: str, force: bool = False) -> Word:
     It is disabled by default to speed-up the overall process, but enabled when
     called from get_and_parse_word().
     """
+    code = re.sub(r"(<!--.*?-->)", "", code, flags=re.DOTALL)
     parsed_sections = find_sections(code, locale)
```

Comment:
Shouldn't  we do that in `cleanup()` directly?

Reply:
Where exactly ?

Reply:
Maybe https://github.com/BoboTiG/ebook-reader-dict/blob/master/wikidict/utils.py#L372?

Reply:
We call `clean` in `process_templates`, so after we have found the sections. The bug here is that the HTML comment hides the section to WTP I believe, so we don't get the section or the section title and we don't process any templates in it.